tcf_core/settings/base.py

```diff
@@ -12,25 +12,27 @@
 """
 
 import os
+
 from django.urls import reverse_lazy
 from django.contrib.messages import constants as messages
 import environ
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
What does this do?

Reply:
`__file__` is a special variable for the current file (i.e. `tcf_core/settings/base.py` in this case). Calling `.dirname()` on the absolute path to this file will return the path to the parent directory (i.e. `tcf_core\settings`). If we do that two more times (so 3 in total), we can the path to the root directory. The original code needed only two `.dirname()` because `settings.py` was in `tcf_core`, but we need one more because the depth increased by 1.